test/node-unit/http.spec.js

```diff
@@ -1,16 +1,24 @@
 'use strict';
 
 var http = require('http');
+var getPort = require('get-port');
 
 var server = http.createServer(function (req, res) {
   res.end('Hello World\n');
 });
 
-server.listen(8888);
-
 describe('http', function () {
+  var port;
+
+  before(function () {
+    return getPort().then(function (p) {
+      port = p;
+      server.listen(port);
+    });
+  });
+
   it('should provide an example', function (done) {
-    http.get({ path: '/', port: 8888 }, function (res) {
+    http.get({ path: '/', port: port }, function (res) {
```

Comment:
Just my two cents' worth, but I don't think we should be adding dependencies to tweak this test. It doesn't seem to be a real unit test of Mocha to begin with -- probably more like an example use case. If it's causing trouble we should stop running it and (if we want to keep it as an example) move it into documentation somewhere instead.